### ISSUE TITLE
Error message when fzf-widgets disabled

### ISSUE THREAD

**Issue by harryxu** [viasite-ansible/ansible-role-zsh#20]
If I remove fzf-widgets bundle in `zsh_antigen_bundles `, will get error message like below:

```
/home/vagrant/.zshrc:49: FZF_WIDGETS_OPTS: assignment to invalid subscript range
```

**Comment by harryxu** [comment]
Line 49 in .zshrc:
```
 46 # fzf-widgets: fzf-change-reset-dir
 47 autoload -Uz chpwd_recent_dirs cdr add-zsh-hook
 48 add-zsh-hook chpwd chpwd_recent_dirs
 49 FZF_WIDGETS_OPTS[insert-history]="--exact"
```

**Comment by popstas** [comment]
Thank you for report, I will fix it at weekend.

**Comment by popstas** [comment]
@harryxu, today is weekend and I reviewed your issue :) Sorry.

I checked code and was very surprised that issue was fixed at 27 May 2017, in 58965f1. A am always using master branch and don't seen this bug.

If you still using role, please upgrade it:
```
ansible-galaxy install viasite-ansible.zsh --force
```
and provision again.
